src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -75,6 +75,16 @@ class ProxyRoutesSpec extends FlatSpec with Matchers with BeforeAndAfterAll with
     }
   }
 
+  it should "set CORS headers in proxy requests" in {
```

Comment:
Previously a custom CORS was only validated in the `/setCookie` tests, which don't proxy through to a backend server. Hence this issue being missed :(